examples/with-supabase/.env.example

```diff
@@ -1,4 +0,0 @@
-# Update these with your Supabase details from your project settings > API
```

Comment:
was this deleted on purpose?
or just renamed & the .env.local isn't tracked so this looks deleted?

Reply:
was a mistake